src/types/types.ts

```diff
@@ -1,83 +1,83 @@
-//block types are strings
-export enum Block {
-  I = "I",
-  J = "J",
-  L = "L",
-  O = "O",
-  S = "S",
-  T = "T",
-  Z = "Z",
-}
+  //block types are strings
```

Comment:
Suggest putting each type/enum into its own file. It makes a lot of files, but it makes the code more navigable.